manager/dispatcher/dispatcher.go

```diff
@@ -603,16 +607,26 @@ func (d *Dispatcher) Tasks(r *api.TasksRequest, stream api.Dispatcher_TasksServe
 					delete(tasksMap, v.Task.ID)
 					modificationCnt++
 				}
-			case <-time.After(eventPausedGap):
 				if modificationCnt > 0 {
```

Comment:
This `if` is not needed here. All the paths to here are after `modificationCnt++`.